web/static/js/efp-runner.js

```diff
@@ -98,7 +98,7 @@ function updateCommitUpdateInfo() {
     const currentCommitDate = document.querySelector('#commit-description .date').innerHTML;
     const playerRepoUrl = document.querySelector('#id_player_repo_url').value;
     if (playerRepoUrl && currentCommitDate) {
-        const githubApiUrl = `${playerRepoUrl}/commits?since=${currentCommitDate}`.replace('github.com', 'api.github.com/repos')
+        const githubApiUrl = `${playerRepoUrl}/commits?since=${currentCommitDate}&sha=master`.replace('github.com', 'api.github.com/repos')
```

Comment:
There is an EFP branch variable in the project settings, `EMBER_EXP_PLAYER_BRANCH`. Do we want to insert that here instead of hard coding?

Reply:
Sure.  I'll make that update now.